config/prow/jenkins/update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+kubectl apply -f jenkins-namespace.yaml 
+kubectl create -f jenkins-pvc.yaml --namespace jenkins
+
+export chart=jenkins/jenkins
+helm upgrade -f jenkins.yaml jenkins $chart  -n jenkins
+
```

Comment:
In what case would we do a `helm upgrade`?
Also, hope namespace and pvc creation would be one time task before doing a `helm install`?